portal-api/src/v1/gef/controllers/eligibilityCriteria.controller.js

```diff
@@ -52,6 +52,6 @@ exports.create = async (req, res) => {
 
 exports.delete = async (req, res) => {
   const collection = await db.getCollection(collectionName);
-  const response = await collection.findOneAndDelete({ version: Number(req.params.version) });
+  const response = await collection.findOneAndDelete({ version: { $eq: Number(req.params.version) } });
```

Comment:
Can we please add `typeof number` validation and remove `Number()`

Reply:
`req.params.version` will be a string as it comes from the request url

Reply:
`req.params.version` will be a string as it comes from the request url